setup.py

```diff
@@ -202,37 +215,47 @@ def move_output(self, ext):
         self.copy_file(source_path, dest_path)
 
 
-version = get_version()
-
 (cc, cxx, mpicc, mpicxx) = find_compilers()
 
 ext_modules = [CMakeExtension("toast._libtoast")]
 
 if mpicxx is not None:
     ext_modules.append(CMakeExtension("toast._libtoast_mpi"))
 
+
+def readme():
+    with open("README.md") as f:
+        return f.read()
+
+
 conf = dict()
 conf["name"] = "toast"
 conf["description"] = "Time Ordered Astrophysics Scalable Tools"
+conf["long_description"] = readme()
+conf["long_description_content_type"] = "text/markdown"
 conf["author"] = "Theodore Kisner, Reijo Keskitalo"
 conf["author_email"] = "tskisner.public@gmail.com"
 conf["license"] = "BSD"
 conf["url"] = "https://github.com/hpc4cmb/toast"
-conf["version"] = version
+conf["version"] = get_version()
 conf["provides"] = "toast"
-conf["python_requires"] = ">=3.4.0"
-conf["install_requires"] = [
-    "cmake",
-    "numpy",
-    "scipy",
-    "healpy",
-    "matplotlib",
-    "pyephem",
-]
+conf["python_requires"] = ">=3.5.0"
```

Comment:
what about dropping 3.5? so there is no worries of using f-strings

Reply:
I hesitate to drop 3.5, but let me check the default python versions on CentOS and the last couple Ubuntu LTS.

Reply:
Ok, CentOS7 (which we are using to build wheels, and so is the oldest glibc version we care about) ships with python 3.6.  Ubuntu 16.04 LTS ships with python 3.5, but that is now 4 years old and there are two LTS releases since then.  So- I am fine restricting to pyton 3.6.  @keskitalo , do you have any opinion on requiring >=3.6?

Reply:
No objections. I use 3.6, 3.7 and 3.8 on different systems.